src/constants.js

```diff
@@ -7,13 +7,29 @@ export const ERRORS = {
 }
 
 export const MESSAGES = {
+  COULD_NOT_WRITE_JSON: locale =>
+    colors.red(`❌ Error: Could not write keys to ${locale}.json.`),
   NO_LOCALE_FILES: pathName =>
-    colors.red(`❌ Error: There are no locale files in "${pathName}."`),
+    colors.red(
+      `❌ Error: There are no locale files in "${pathName}. \n Only base locales are considered. (i.e. LOCALE.json)`
+    ),
   NO_LOCALE_FOLDER: pathName =>
-    colors.red(`❌ Error: There is no locales folder in "${pathName}."`),
+    colors.red(`❌ Error: There is no locales folder in "${pathName}.`),
```

Comment:
Wouldn't it be better to use "messages" when referring to "locales folder"? Something like "There is no messages folder in ${pathName}"?

Reply:
Nice, I'll do that.